### PR TITLE
Mailchimp integration

### PR DESCRIPTION
**Pull request by xdzurman**
depends on #496

------NOTE------
when testing, Mailchimp API KEY is required. I cannot commit mine to github, since it grants account access. PM me for it.

### CODE REVIEW

File: app/frontend/components/pages/staking/stakingPage.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            type: 'email',
+            placeholder: 'Enter your email to get notified',
+            value: email,
+            required: true,
+            onInput: this.updateEmail,
+          }),
+          h(
+            'button',
+            {
+              onClick: this.handleSubmit.bind(this),
```
Comment (PeterBenc) [inline app/frontend/components/pages/staking/stakingPage.js]: All the other methods are bound in the constructor, maybe I would bind this one too, just to stay consistent :).

File: app/frontend/components/common/contactForm.js [hunk 2]
```diff
@@ -55,6 +59,13 @@ class ContactForm extends Component {
             method: 'POST',
             target: '_blank',
             action: `//formspree.io/${ADALITE_CONFIG.ADALITE_SUPPORT_EMAIL}`,
+            onSubmit: () => {
+              this.setState({sumbitted: true})
```
Comment (PeterBenc) [inline app/frontend/components/common/contactForm.js]: Maybe you could create a method which changes the state of the the contactForm. It's just a detail but we dont use setState method inside render method anywhere in the project, so again just to stay consistent :).

File: app/frontend/components/pages/staking/stakingPage.js (hunk 1 above)
Comment (xdzurman) [reply app/frontend/components/pages/staking/stakingPage.js]: fixed, originally it was a workaround inspired by /reactconf

File: app/frontend/components/common/contactForm.js (hunk 2 above)
Comment (xdzurman) [reply app/frontend/components/common/contactForm.js]: This is the code from the issue branch that this branch depends on so it's not mine, can you put this comment there so the author decides? :)
-> https://github.com/vacuumlabs/adalite/pull/497

**Pull request merged**